File: code-refactoring/long-method/C#/VSCode/long-method/AccountService.cs
public class AccountService
{
    public Account HandeResponse(AccountResponse accountResponse)
    {
        if (accountResponse.Id == null)
        {
            throw new ArgumentException("Id is required");
        }
        if (accountResponse.Name == null)
        {
            throw new ArgumentException("Name is required");
        }
        if (accountResponse.Email == null)
        {
            throw new ArgumentException("Email is required");
        }
        if (accountResponse.Phone == null)
        {
            throw new ArgumentException("Phone is required");
        }

        Account account = new Account();
        account.Id = accountResponse.Id;
        account.Name = accountResponse.Name;
        account.Email = accountResponse.Email;
        account.Phone = accountResponse.Phone;

        return account;
    }
}

File: code-refactoring/long-method/java/src/main/java/com/epam/aicode/java/AccountService.java
package com.epam.aicode.java;

public class AccountService {

   public Account handleResponse(AccountResponse accountResponse) {
       if (accountResponse.getId() == null) {
           throw new IllegalArgumentException("Id is required");
       }
       if (accountResponse.getName() == null) {
           throw new IllegalArgumentException("Name is required");
       }
       if (accountResponse.getEmail() == null) {
           throw new IllegalArgumentException("Email is required");
       }
       if (accountResponse.getPhone() == null) {
           throw new IllegalArgumentException("Phone is required");
       }

       Account account = new Account();
       account.setId(accountResponse.getId());
       account.setName(accountResponse.getName());
       account.setEmail(accountResponse.getEmail());
       account.setPhone(accountResponse.getPhone());

       return account;
   }
}
